examples/BroadcastingEchoServer.cpp

```diff
@@ -10,6 +10,7 @@ int main() {
     uWS::App().ws<PerSocketData>("/*", {
         /* Settings */
         .compression = uWS::SHARED_COMPRESSOR,
+        .compressorOptions = {},
```

Comment:
This could be dedicatedCompressorOptions, it only applies to dedicated compressors

Reply:
Yes you are right - it's a bit long - but I don't have a better descriptive name.

Should I change it?